observatory-dags/observatory/dags/workflows/geonames_telescope.py

```diff
@@ -152,7 +148,7 @@ def __init__(
         schedule_interval: str = "@weekly",
         dataset_id: str = "geonames",
         schema_folder: str = default_schema_folder(),
-        source_format: str = SourceFormat.CSV,
```

Comment:
Mmm I see, thanks for picking that up.
I thought if I would set it the type hint to SourceFormat it would limit the allowed values to those of the SourceFormat class, but that is not true of course.

I've just changed the types to 'str' for now.
There are 6 different source formats atm, so including them all in a Literal type directly would be too long.
Maybe we could create a new type hint class/object and import this everywhere, but I'm not sure if it is worth the effort, the allowed source formats may change over time as well.